src/component/form/TextInputElement/TextInputElement.tsx

```diff
@@ -478,3 +474,11 @@ export default class TextInputElement extends React.Component<TextInputProps, Te
         );
     }
 }
+
+const mapStateToProps = ({ typeConfig, resultNames }: ReduxState) => ({ typeConfig, resultNames });
+
+const ConnectedTextInputElement = connect(mapStateToProps, null, null, { withRef: true })(
```

Comment:
we use the `withRef` property to indicate to `connect` that we're passing a ref to the wrapped component